lib/workspace.rb

```diff
@@ -0,0 +1,65 @@
+require 'httparty'
+require 'dotenv'
+require_relative './user'
+require_relative './channel'
+
+# response = HTTParty.get('https://slack.com/api/channels.list?token=xoxp-my-great-key&pretty=1')
+# puts response.body, response.code, response.message, response.headers.inspect
+
+# https://slack.com/api/channels.list?token=xoxp-my-great-key&pretty=1
+class Workspace
+  include HTTParty
+  base_uri 'slack.com/api'
+
+  def initialize(api_token)
+    @options = { query: { token: api_token, pretty: 1 } }
+  end
+
+  def channels
+    response = self.class.get('/channels.list', @options).parsed_response
+    raise 'Failed to get channels' unless response['ok']
+
```

Comment:
When you raise an exception you should provide an exception type! Otherwise you get a `RuntimeError`, which is pretty generic. Defining a custom exception type like `SlackApiError` might be a good choice here.